sdk/ai/azure-ai-agents/src/main/java/com/azure/ai/agents/AgentsClientBuilder.java

```diff
@@ -396,6 +418,36 @@ private String getUserAgent() {
         return UserAgentUtil.toUserAgentString(applicationId, sdkName, sdkVersion, configuration);
     }
 
+    private HttpClient getOpenAIHttpClient() {
+        if (this.httpClient == null) {
+            return null;
+        }
+        List<HttpPipelinePolicy> orderedPolicies = getOrderedCustomPolicies();
```

Comment:
What happens if the user only configures custom policies but not the HTTP client? In this scenario, the policies are ignored and the default HTTP client will be used.